content/_data/changelogs/weekly.yml

```
@@ -5985,6 +5985,47 @@
 
   # pull: 4438 (PR title: Removed unnecessary semicolons and unused imports)
 
+- version: '2.218'
+  date: 2020-01-26
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4454
+    issue: 60857
+    authors:
+    - jglick
+    references:
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.8
+        title: Winstone 5.8 release notes - regression in TODO
+    message: |-
+      Fixes a <code>KeyStores with multiple certificates are not supported</code> error from Jetty when passing certain kinds of certificates, such as domain wildcards (regression in 2.217)
+  - type: rfe
+    category: rfe
+    pull: 4436
+    authors:
+    - daniel-beck
+    message: |-
+      Add the boolean <code>disabled</code> flag to applicable jobs' REST API output for classic project types like Freestyle, Maven, or Multi-configuration (Matrix).
+  - type: rfe
+    category: rfe
+    pull: 4452
+    issue: 60821
+    authors:
+    - oleg-nenashev
+    references:
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-5.7
+        title: Winstone 5.7 release notes
+    message: |-
+      Internal: Winstone 5.7: Change the Jetty thread pool name to "Jetty (winstone)"
+  - type: bug
+    category: bug
+    pull: 4452
+    issue: 57888
+    authors:
+    - oleg-nenashev
+    message: |-
+      Internal: Winstone 5.7: Fix support of system logging customization (regression in 2.177)
```

```suggestion
      Internal: Winstone 5.7: Fix support of system logging customization (regression in 2.177)
```

It is not internal, it is a user-facing bug. I suggest to reorder it accordingly

---

I moved it higher in the list with https://github.com/jenkins-infra/jenkins.io/pull/2801/commits/f17f17e24bdd9b19458a27b337a2e3f5e6428a97 and I removed the "Internal:" text from the summary.  Let me know if you feel it should be placed somewhere different.